src/components/Toggle/Toggle.stories.mdx

```diff
@@ -0,0 +1,180 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { HStack, VStack, Grid } from "@chakra-ui/react";
```

Comment:
Add `Box` here.

Reply:
For some reason auto-import doesn't work on mdx files for me - have you found a VSCode extension that will help with this?

Reply:
I haven't. My vscode also doesn't tell me if there are syntax or variable issues in the file so it's a bit rough at times.